core/src/storage_stage.rs

```diff
@@ -218,58 +220,57 @@ impl StorageStage {
         }
     }
 
-    fn send_tx(
+    fn send_transaction(
         cluster_info: &Arc<RwLock<ClusterInfo>>,
-        tx: &mut Transaction,
+        transaction: &mut Transaction,
         exit: &Arc<AtomicBool>,
         keypair: &Arc<Keypair>,
         account_to_create: Option<Pubkey>,
     ) -> io::Result<()> {
-        if let Some(leader_info) = cluster_info.read().unwrap().leader_data() {
-            let mut client = mk_client_with_timeout(leader_info, Duration::from_secs(5));
+        let node_info = cluster_info.read().unwrap().my_data();
```

Comment:
`s/leader_data/my_data/` on this line is really the only functional change in this PR.   

The rest of the patch is just code shifting around and my desire to purge the codebase of "tx" (like does this mean "transaction" or "transmit"...we use "tx" for both!)